### ISSUE TITLE
Brightness does not work

### ISSUE THREAD

**Issue by FamiliarBreakfast** [jmontleon/pixelbook-fedora#59]
The backlight can't be changed in brightness. only turned on or off

**Comment by jmontleon** [comment]
I need a lot more information than that. It's working great for me, and as far as I know for others.

What distribution and release? What kernel? What MrChromebox firmware version? If Fedora have you follow the setup here? What else have you tried?

**Comment by jmontleon** [comment]
Please feel free to reopen this if you can provide more information. Thank you.

**Comment by FamiliarBreakfast** [comment]
Sorry about the late response, I totally forgot I made this! It's a Pixelbook Eve running kernel 6.1.14-200, Fedora 37, Sway WM. Yes, I followed the setup here, mostly;
Except, I'm only running RW_LEGACY firmware, version Google_Eve.9584.230.0. The more I think about it, I assume this is the issue? Or perhaps it's Wayland? I can try with X and report back.

I've tried several brightness utilities, as well as just directly through the driver. When set to 0, the screen turns off, but the system is still functional. Anything above 0 sets the backlight to max brightness.  
I've tried setting the kernel parameter `enable_dpcd_backlight`, it doesn't change anything.

Everything else works fine, including the keyboard backlight, though I understand this is a different system.
